src/components/Button/Button.tsx

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames';
+import { ReactNode } from 'react';
+
+export enum ButtonTypes {
+  Primary = 'primary',
+  HasIcon = 'has-icon',
+}
+
+interface ButtonProps {
+  icon?: ReactNode;
+  onClick?: () => void;
+  text?: string;
+  variant?: ButtonTypes;
+  width?: string;
+}
+
+const Button = ({
+  icon,
+  onClick,
+  text,
+  variant,
+  width = 'w-full',
+}: ButtonProps): JSX.Element => {
+  const btnClass = classNames(
+    'rounded-lg flex align-center gap-3 px-[1rem] py-[.75rem] border-dark-orange font-semibold text-white text-center',
```

Comment:
use provided tailwind measurements `px-[1rem]` to `px-4`, and `py-[.75rem]` to `py-3`